btree/btree.h

```diff
@@ -316,6 +316,24 @@ void m0_btree_op_fini(struct m0_btree_op *bop);
 void m0_btree_op_credit(const struct m0_btree_op *bt,
 			struct m0_be_tx_credit *cr);
 
```

Comment:
Not using m0_btree_op_credit as  we are calculating credits before calling m0_btree_put/ m0_btree_update etc, and m0_btree_op will get filled in m0_btree_put this routine so operation type will not be available at time of calling credit calculation if we are using m0_btree_op_credit . Therefore, adding separate interface for credit calculation for each operation.